frontend/packages/schema-editor/src/components/SchemaEditor/SchemaEditor.tsx

```diff
@@ -19,8 +19,6 @@ export const SchemaEditor = () => {
   const { data: user } = useUserQuery();
   const moveProperty = useMoveProperty();
   const addReference = useAddReference();
-
-  if (schemaModel.isEmpty()) return null;
```

Comment:
Maybe we can add a small test to check that the root header still presented after deleting all elements. 🙂